server/app/services/call_service.py

```
@@ -1662,4 +1667,281 @@ async def get_service_advisors(
+                    detail=f"Organization not found for playground rooftop {rooftop_uuid}"
+                )
+            
+            logger.info(f"Using PLAYGROUND environment - Org: {organization.name} ({organization.id}), Rooftop: {rooftop.name} ({rooftop.id})")
+            
+        # Validate and format phone number (common for both local and playground)
+        clean_phone = re.sub(r'[\s\-\(\)]', '', customer_phone)
+        if not re.match(r'^\+(\d{1,4})(\d{7,14})$', clean_phone):
+            raise HTTPException(
+                status_code=400,
+                detail="Invalid phone number format"
+            )
```

Ruff (0.14.7)

Abstract `raise` to an inner function

(TRY301)